Description: Explain failure
 Due to vinyl-fs version, watch() function isn't available.
Author: Xavier Guimard <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/862887
Forwarded: not-needed
Last-Update: 2019-02-13

--- a/index.js
+++ b/index.js
@@ -36,6 +36,9 @@
       'a function (optionally generated by using gulp.parallel ' +
       'or gulp.series)');
   }
+  if(!vfs.watch) {
+    throw Error("Gulp.watch is disabled in this version of gulp. See /usr/share/doc/gulp/README.Debian");
+  }
 
   if (typeof opt === 'function') {
     task = opt;
